fhevm/params.go

```diff
@@ -67,116 +67,139 @@ type GasCosts struct {
 func DefaultGasCosts() GasCosts {
 	return GasCosts{
 		FheAddSub: map[FheUintType]uint64{
+			FheUint4:  84000 + AdjustFHEGas,
```

Comment:
is it normal that there is no gas difference between euint4 and euint8? I was expecting euint4 to be cheaper.

Reply:
Yes because I don't have up-to-date benchmark for FheUint4. I'll do a PR to update all gas regarding bool and FheUint4 later